tests/functional/string_test.py

```diff
@@ -98,6 +98,17 @@ def test_setex():
     assert(ret == 1)
     assert_raise(redis.RedisError, conn.setex, "foo", "bar", "invalid")
 
+def test_psetex():
+    key = "test_psetex"
+    conn = get_redis_conn()
+    ret = conn.psetex(key, "bar", 1000*1024)
```

Comment:
need to change the parameter order: psetex {time_ms} {key}

Reply:
Thanks @karelrooted Tbn, i didn't run this test, initially, i copied test form `test_setex`, it seems `psetex` is different with `setex`, `setex` syntax is `setex key value time`, but `psetex` still respect redis syntax.